presenters/booking_presenter.rb

```diff
@@ -0,0 +1,73 @@
+class BookingPresenter
+  TICKETS_COUNT_LIMIT = 5
+
+  delegate :title,  to: :department,  prefix: true
+  delegate :title,  to: :arrival,     prefix: true
+  delegate :id,     to: :route,       prefix: true
+
+  delegate :first_class_price,  to: :config
+  delegate :second_class_price, to: :config
+  delegate :price_coefficient,  to: :route
+
+  def initialize(params)
+    @params = params
+  end
+
+  def department
+    @department ||= City.find_by(id: route&.start_id)
```

Comment:
Try to position methods in order of their dependencies. For example, `route` depends only on params so it should go first. `arrival, department` and `train` depend on the route so they should be positioned below `route` definition. It's not a rule or anything but it's easier to read, at least for me :)